assignment2/src/views/GameView.vue

```diff
@@ -0,0 +1,10 @@
+<script setup lang="ts">
+    import Gameboard from '../components/GameBoard.vue'
+</script>
+
+<template>
+    <!-- <Gameboard :guess="$root!.$data.guesses" :colors="$root!.$data.letterColor"></Gameboard> -->
+</template>
+
+<style>
```

Comment:
Unsure if work is still in progress, but the GameView does not have content. This project is missing multiple components and scripts that were implemented in class.

Reply:
The commented out component fits the visual aspect of the game board just is missing the logic. But because of the attempt at referencing theme colors causing an error in the pipeline (but running fine on localhost) I cut it out so the site would be at least running for the PR. I got too hooked on making it fully functional as I wanted it to be and not just making the most viable product that suited the requirements so it's a bit of a mess but hopefully I'll get it together for the final submit.